zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/Constants.java

```diff
@@ -26,6 +26,8 @@
 
   public static final String ZEPPELIN_INTERPRETER_HOST = "zeppelin.interpreter.host";
 
+  public static final String ZEPPELIN_PRECODE_PROPERTY_KEY = "zeppelin.interpreter.precode";
```

Comment:
in planning for the future add this property to another interpreters (PySpark, SparkR)